docs/source/point-evaluation.rst

```diff
@@ -98,6 +100,235 @@ in parallel. There is no special API, but there are some restrictions:
+:math:`\delta` is a dirac delta distribition can therefore be written in
+Firedrake using :func:`~.VertexOnlyMesh` and :func:`~.interpolate` as
+
+.. literalinclude:: ../../tests/vertexonly/test_vertex_only_manual.py
+   :language: python3
+   :dedent:
+   :lines: 50-64
+
+Interacting with external point data
+------------------------------------
+
+.. warning::
+
+   The examples given in the following section use an internal Firedrake API
+   ``Function.dat.data`` which could change in the future.
+
+.. warning::
+
+   The examples given in the following section will not work in parallel. A
+   parallel safe way of interacting with external data will be available soon.
+
+Any set of points with associated data in our domain can be expressed as a
+P0DG function on a :func:`~.VertexOnlyMesh`.
+
+.. code-block:: python3
+
+   vom = VertexOnlyMesh(parent_mesh, point_locations_from_elsewhere)
+   P0DG = FunctionSpace(vom, "DG", 0)
+   y_pts = Function(P0DG).dat.data[:] = point_data_values_from_elsewhere
+
+We can use :func:`~.interpolate` to interact with this data to, for example,
+compare a PDE solution with the point data. The :math:`l_2` error norm
+(euclidean norm) of a function :math:`f` (which may be a PDE solution)
+evaluated against a set of point data :math:`\{y_i\}_{i=0}^{N-1}` at points
+:math:`\{x_i\}_{i=0}^{N-1}` is defined as
+
+.. math::
+
+   \sqrt{ \sum_{i=0}^{N-1} (f(x_i) - y_i)^2 }.
+
+We can express this in Firedrake as
+
+.. code-block:: python3
+
+   error = sqrt(assemble((interpolate(f, P0DG) - y_pts)**2*dx))
+
+   # or equivalently
+   error = errornorm(interpolate(f, P0DG), y_pts)
+
+.. _tolerance:
+
+Mesh tolerance
+--------------
+
+If points are outside the mesh domain but ought to still be found a
+``tolerance`` parameter can be set. The tolerance is relative to the size of
+the mesh cells and is a property of the mesh itself
+
+.. literalinclude:: ../../tests/vertexonly/test_vertex_only_manual.py
+   :language: python3
+   :dedent:
+   :lines: 70-90
+
+Keyword arguments
+~~~~~~~~~~~~~~~~~
+
+Alternatively we can modify the tolerance by providing it as an argument to the
+vertex-only mesh. This will modify the tolerance property of the parent mesh.
+
+.. warning::
+
+   To avoid confusion, it is recommended that the tolerance be set for a mesh
+   before any point evaluations are performed, rather than making use of these
+   keyword arguments.
+
+.. literalinclude:: ../../tests/vertexonly/test_vertex_only_manual.py
+   :language: python3
+   :dedent:
+   :lines: 100-117
+
+Note that since our tolerance is relative, the number of cells in a mesh
+dictates the point loss behaviour close to cell edges. So the mesh
+``UnitSquareMesh(5, 5, quadrilateral = True)`` will include the point
+:math:`(1.1, 1.0)` by default.
+
+If a mesh tolerance changes where we already have an immersed vertex-only mesh,
+such that the new tolerance ought to cause points to disappear, they will not.
```

Comment:
I don't quite understand this paragraph.

Reply:
It's saying that changing mesh tolerance only effects point location when you create a new VOM

Reply:
Would it be clearer if I explicitly stated that?

Reply:
I think so.

Reply:
Can you take a look at my proposed diff and see if it's alright?

Reply:
Yep I'm happier with that. Thanks. If you commit those diffs I'll approve the PR.